docker-compose.yml

```diff
@@ -8,6 +8,22 @@ services:
     working_dir: /home/node/app
     volumes:
     - ./:/home/node/app
-    command: npm run serve -- --port 8081
+    command: npm run serve -- --port 8081 --host ui
     ports:
       - 8081:8081
+    healthcheck:
+      test: curl --fail http://ui:8081 || exit 1
```

Comment:
Question: Is `|| exit 1` needed when `curl --fail` is used? I would think this already exits non-zero on failure, so there's no need to repeat this.